ckanext/multilang/i18n/ckanext-multilang.pot

```diff
@@ -0,0 +1,60 @@
+# Translations template for ckanext-multilang.
+# Copyright (C) 2018 ORGANIZATION
+# This file is distributed under the same license as the ckanext-multilang
+# project.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2018.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: ckanext-multilang 0.0.1\n"
+"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
+"POT-Creation-Date: 2018-04-10 19:06+0200\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: Babel 2.4.0\n"
+
+#: ckanext/multilang/plugin.py:333 ckanext/multilang/plugin.py:342
+msgid "Language"
+msgstr ""
+
+#: ckanext/multilang/plugin.py:343
+msgid "Enter language code"
+msgstr ""
+
+#: ckanext/multilang/plugin.py:344
+msgid "Set language for which this resource will be visible"
+msgstr ""
+
+#: ckanext/multilang/controllers/package.py:175
+#: ckanext/multilang/controllers/package.py:262
+#, python-format
+msgid "Unauthorized to read package %s"
```

Comment:
shouldnt "{}" be used for params?

Reply:
Those messages are generated from code not covered by this PR: https://github.com/cezio/ckanext-multilang/blame/11_localized_resource_urls/ckanext/multilang/controllers/package.py#L165-L175